BedrockServer.cpp

```diff
@@ -538,18 +527,13 @@ void BedrockServer::sync()
                             SERROR("peekCommand (" << command->request.getVerb() << ") returned invalid result code: " << (int)result);
                         }
 
-                        // If we just started a new HTTPS request, save it for later.
+                        // If this command attempted an HTTP request, kill it.
                         if (command->httpsRequests.size()) {
-                            waitForHTTPS(move(command));
-                            // TODO:
-                            // Move the HTTPS loop into the worker, so that the worker can poll on its own requests.
-                            // This is the first step toward linear workers that run start->finish without being
-                            // interrupted and bounced back and forth between a bunch of queues.
-                            //
-                            // The follow-up to that is to allow direct escalations from workers to leader, which can
-                            // be done as a simple HTTPS request for the exact same command.
-
-                            // Move on to the next command until this one finishes.
+                            SWARN("Killing command " << command->request.methodLine << " that attempted HTTPS request in sync thread.");
```

Comment:
What is the scenario that this code would get run and what would happen with the command in that case?
I think the answer is: this should run on the worker and would probably never happen in the sync thread anyway, but if it happens (which is possible), the command would fail to run?

Reply:
I haven't been able to actually make this happen in testing, I don't think there's a reasonable path to get it to happen, but if it did, the command would error, yes.